pysite/constants.py

```diff
@@ -20,3 +21,7 @@ class ValidationTypes(Enum):
 ADMIN_ROLE = 267628507062992896
 MODERATOR_ROLE = 267629731250176001
 HELPER_ROLE = 267630620367257601
+
+CLIENT_ID = environ.get('CLIENT_ID', '')
+CLIENT_SECRET = environ.get('CLIENT_SECRET', '')
+SCOPE = 'identify email'
```

Comment:
Too generic - can we prefix all of these with, eg, `DISCORD_OAUTH`? Env vars too.

Reply:
Alrighty

Reply:
Did you get around to looking into this?